src/test/store/fixtures/profiles.js

```diff
@@ -24,13 +24,30 @@ export function getProfileWithMarkers(markers: TestDefinedMarkers): Profile {
     data: [],
     length: 0,
   };
+  const samples = {
+    time: [],
+    responsiveness: [],
+    stack: [],
+    rss: [],
+    uss: [],
+    length: 0,
+  };
+
   markers.forEach(([name, time, data]) => {
     markersTable.name.push(stringTable.indexForString(name));
     markersTable.time.push(time);
     markersTable.data.push(data);
     markersTable.length++;
+
+    // trying to get a consistent profile with a sample for each marker
+    samples.time.push(data.startTime, data.endTime);
+    samples.length++;
   });
-  profile.threads.push(Object.assign({}, thread, { markers: markersTable }));
+
+  samples.time.sort();
+  profile.threads.push(
+    Object.assign({}, thread, { markers: markersTable, samples })
+  );
```

Comment:
without any sample, the display range goes from +Infinity to -Infinity (not the opposite !)... which is wrong of course.

Another possible fix is to consider markers in addition to samples in https://github.com/devtools-html/perf.html/blob/6b7f9fb0ccaa176b84e9787faa349099064b92e6/src/profile-logic/profile-data.js#L151, please tell me if you'd prefer this.